engine/collection/ingest/engine.go

```diff
@@ -367,7 +368,12 @@ func (e *Engine) ingestTransaction(
 func (e *Engine) propagateTransaction(log zerolog.Logger, tx *flow.TransactionBody, txCluster flow.IdentitySkeletonList) {
 	log.Debug().Msg("propagating transaction to cluster")
 
-	err := e.conduit.Multicast(tx, e.config.PropagationRedundancy+1, txCluster.NodeIDs()...)
+	msg, err := messages.InternalToMessage(tx)
+	if err != nil {
+		e.log.Error().Err(err).Msg("failed to convert event to message")
```

Comment:
```suggestion
		e.log.Error().Err(err).Msg("failed to convert event to message")
		return
```
we cannot continue to broadcast if the conversion has failed